core/domain/user_services_test.py

```diff
@@ -874,6 +1129,36 @@ def test_removing_default_user_role_raises_exception(self):
         with self.assertRaisesRegex(Exception, error_msg):
             user_services.remove_user_role(user_id, feconf.ROLE_ID_FULL_USER)
 
+    def test_is_user_registered_with_existing_user_id_returns_true(self):
```

Comment:
Perhaps change "with" to "for", for parity with the next test.

Reply:
Done